binding.gyp

```diff
@@ -70,10 +83,25 @@
           'outputs': [
             '<@(examples_idl_output_files)',
           ],
-          'action': [
-            '<@(PRODUCT_DIR)/../../bootstrap/command/node',
-            '<@(PRODUCT_DIR)/generator/main.js',
-            '<@(_inputs)'
+          'conditions': [
+            ['OS!="win"',
+            {
+              'action': [
+                '<@(PRODUCT_DIR)/../../bootstrap/command/node',
+                '<@(PRODUCT_DIR)/generator/main.js',
+                '<@(_inputs)'
+              ],
+            }],
+            ['OS=="win"',
+            {
+              'action': [
+                '<@(PRODUCT_DIR)/../../third_party/node/node.exe',
+                '<@(PRODUCT_DIR)/generator/main.js',
+                # FIXME :: didn't find template files when nunjucks.render executed.
```

Comment:
Could you file a bug in issue tab and then use a FIXME(your id) format?
```# FIXME(hwanseung): Didn't find template files ....```

Reply:
done